frontend/test/metabase/scenarios/question/reproductions/17910-revision-history-update.js

```diff
@@ -0,0 +1,33 @@
+import { restore, openOrdersTable, modal } from "__support__/e2e/cypress";
+
+export function issue17910() {
+  describe("issue 17910", () => {
+    beforeEach(() => {
+      restore();
+      cy.signInAsNormalUser();
+    });
+
+    it("revisions should work after creating a question without reloading", () => {
+      openOrdersTable();
+      cy.intercept("POST", `/api/card`).as("card");
+      cy.findByText("Save").click();
+      modal().within(() => {
+        cy.findByText("Save").click();
+      });
+      cy.wait("@card");
+      modal().within(() => {
+        cy.findByText("Not now").click();
+      });
```

Comment:
We can (and should) use API for this part. It makes no difference for the revision history, but it makes test faster and more resilient.

Reply:
```js
cy.createQuestion(
  {
    name: "17910",
    query: { "source-table": ORDERS_ID },
  },
  // This part already does the intercepting and waiting for you under the hood
  { visitQuestion: true },
);
```

Reply:
I'll give this a try, but I'm pretty sure that the bug relies on the events that occur from creating the question via the UI. It's the redux events firing that puts us in a bad state

Reply:
I tried to repro with creating the question via the API but it didn't reproduce the bug.

Reply:
In that case, go for it :)
Thanks for going the extra mile!